examples/connectivity/plot_mne_inverse_envelope_correlation.py

```diff
@@ -0,0 +1,84 @@
+"""
+=============================================
+Compute envelope correlations in source space
+=============================================
+
+Compute orthogonal envelope correlations of activity in source space
+using resting state CTF data.
+"""
+# Authors: Eric Larson <larson.eric.d@gmail.com>
+#          Sheraz Khan <sheraz@khansheraz.com>
+#          Denis Engemann <denis.engemann@gmail.com>
+#
+# License: BSD (3-clause)
+
```

Comment:
Getting ValueError on running this example:

"/Users/dengeman/mne_data/MNE-brainstorm-data/bst_resting/MEG/bst_resting/subj002_spontaneous_20111102_01_AUX.ds"

Reply:
Your dataset is out of date